frontend/public/style/mixin/_line-clamp.scss

```
@@ -0,0 +1,6 @@
+@mixin co-line-clamp($lines) {
```

Add a default value so `$lines` is optional
```suggestion
@mixin co-line-clamp($lines: 3) {
```